### PR TITLE
New license: MMIXWare

### PR DESCRIPTION
**Pull request by xsuchy**
Fixes: #2028

Submitted with the name changed to what @swinslow suggested.

### CODE REVIEW

File: src/MMIXWare.xml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license isOsiApproved="false" licenseId="MMIXware"
+   name="MMIXware License" listVersionAdded="3.22">
+      <crossRefs>
+         <crossRef>https://gitlab.lrz.de/mmix/mmixware/-/blob/master/boilerplate.w</crossRef>
+      </crossRefs>
+      <text>
+         <list>
+            <item>
```
Comment (jlovejoy) [inline src/MMIXWare.xml:10]: not sure why the list and item tags got in here, but not needed

File: src/MMIXWare.xml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+               <p>
+                  copyright 1999 Donald E. Knuth This file may be freely copied and
```
Comment (jlovejoy) [inline src/MMIXWare.xml:12]: copyright notice needs to be enclosed in copyrightText tag

File: src/MMIXWare.xml (hunk 1 above)
Comment (xsuchy) [reply src/MMIXWare.xml:10]: SPDX online tools generated that. I will remove it.

File: src/MMIXWare.xml (hunk 2 above)
Comment (xsuchy) [reply src/MMIXWare.xml:12]: I checked how other licenses in src/ do that and I do that similarly. Tell me if needs to be done differently.

**Comment by xsuchy** [comment]
Updated.

**Comment by jlovejoy** [comment]
@swinslow @goneall - wondering if either of you have opinions on the "genercizing" of the copyright notice. I know some people have asked for that on specific licenses and we have sometimes obliged, but do we want to do that preemptively?

in this case, .txt file has name of copyright holder from original source and .xml file has YEAR, NAME - it's passing tests, but I sort am incline this should match b/w the files in any case?  I know the copyright notice isn't "matched", but for visualization of license, it might be good if we had a definitive policy (and documented it in the xml guidance doc)??

**Comment by goneall** [comment]
> wondering if either of you have opinions on the "genercizing" of the copyright notice. I know some people have asked for that on specific licenses and we have sometimes obliged, but do we want to do that preemptively?

No opinion on the policy itself, but +1 on having a policy and documenting it.  As long as the copyright text is wrapped in the `<copyrightText>` element tag, the license list publisher will create an "alt" text so that any text will match the copyright text itself.

**Comment by jlovejoy** [comment]
thanks @goneall - I'm going to boot the policy decision to a later time as I'd like to get this merged.

In this case, I think we should just match the text file but using "1999 Donald E. Knuth" in the copyright notice (which is contained in the copyrightText tag only) in the .xml file as well. It's a project-specific license as far as we know. If we later find other uses with a different project (and copyright holder) we can then add markup and genericize the copyright notice at that time

**Comment by jlovejoy** [comment]
@xsuchy - I just went ahead and fixed that one thing myself. not sure if that's proper git etiquette, but trying to just get stuff done... :)

**Pull request merged**